### ISSUE TITLE
Web administration page is available for only, a short period of time.

### ISSUE THREAD

**Issue by sitle** [pritunl/pritunl#310]
I just updated pritunl to version v1.21.938.10 on Ubuntu 14.04.

I have this log : 

```
Traceback (most recent call last):
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/subscription.py", line 38, in update
    timeout=max(settings.app.http_request_timeout, 10),
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/utils/request.py", line 68, in get
    return _request('GET', url, **kwargs)
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/utils/request.py", line 65, in _request
    raise httplib.HTTPException(error)
HTTPException: urlopen() got an unexpected keyword argument 'context'
[guarded-plateau-1521][2016-04-07 03:13:11,722][ERROR] Failed to check subscription status
```

I don't know what's going on. Only the web interface bugs, everything else works smoothly.

**Comment by teamfighter** [comment]
Confirming, and if you will try to re-install app, it will be totally broken and unusable.

**Comment by hardzink** [comment]
![photo230588805939308739](https://cloud.githubusercontent.com/assets/18180219/14358490/b80a0a7a-fd1f-11e5-9761-d80fda5c6820.jpg)
 
i got this problem after update this version

**Comment by hardzink** [comment]
![pritunl](https://cloud.githubusercontent.com/assets/18180219/14358524/e761eba8-fd1f-11e5-9fbc-a99c8f65ac7b.jpg)


i got this problem after update this version

**Comment by hardzink** [comment]
![untitled](https://cloud.githubusercontent.com/assets/18180219/14358667/8e140fbc-fd20-11e5-841e-86e6c1351a07.png)


https://ipaddress:443

**Comment by zachhuff386** [comment]
Fixed in v1.21.938.80
